inspirehep/modules/workflows/workflows/article.py

```diff
@@ -375,7 +366,8 @@ class Article(object):
                         # TODO: once legacy is out, this should become
                         # unconditional, and remove the SEND_TO_LEGACY_AND_WAIT
                         # steps
-                        IF_NOT(in_production_mode, [store_record]),
```

Comment:
Should this be happening already? Before Labs is the master of the data, do we want already to save locally?